resources/shared_translations.txt

```diff
@@ -4104,6 +4104,12 @@
 "operations.lannan-varastointi.desc" "sv" "Denna blankett används för att till kommunens miljöskyddsmyndighet anmäla om <p><ol><li>Sådan avvikelse från den lagringsvolym, som en gård där nötkreatursgödsel lagras enligt statsrådets förordning  (931/2000) (nitratförordningen) om begränsning av utsläpp i vattnen av nitrater från jordbruket  förutsätts ha (§4 mom1), som avses i §4 mom3. (På blanketten ifylls uppgifter om anmälaren samt data om antalet djur och omständigheter som påverkar gödselhanteringen. Beträffande undantagssättet fyller du i någon/några av de relevanta punkterna A, B, C tai D. Om det finns fler än en mottagare av typ A-C ska uppgifterna om varje mottagare fyllas i separat.)</li><li>Annan än i punkt 1 avsedd fjärrlagring av gödsel i batteri enligt bestämmelserna i nitratförordningen §3 och bilaga 1 (På blanketten fyller du i uppgifterna om anmälaren samt data om antalet djur på gården och om omständigheter som påverkar fjärrlagringen av gödsel i batteri. Beträffande batterianläggningen fyller du i punkt D.)</li></ol></p><p>Anmälan ska göras minst en månad innan meningen är att vidta åtgärder som syftar till i punkt 1 avsedd avvikelse eller i punkt 2 avsedd annan gödsellagring i batteri.</p><p>Anmälan ska förnyas alltid när planerade åtgärder skulle avvika från de åtgärder som föreslagits i den tidigare gjorda anmälan.</p><p>Anmälan ska göras av den lantbruksföretagare som avser att avvika från nitratförordningens krav på lagringsvolym för gödsel (punkt 1) eller som har för avsikt att börja med annan lagring av gödsel i batteri (punkt 2).</p>"
+"operations.leirintaalueilmoitus" "en" "Camping area announcement"
+"operations.leirintaalueilmoitus" "fi" "Leinrintäalueilmoitus"
+"operations.leirintaalueilmoitus" "sv" "Anmälan om campingplats"
+"operations.leirintaalueilmoitus.desc" "en" "<p></p>"
+"operations.leirintaalueilmoitus.desc" "fi" "<p></p>"
+"operations.leirintaalueilmoitus.desc" "sv" "<p></p>"
```

Comment:
Nämä ovat varmaan jotain placeholdereita ja tarkoitus ei ole vain tuottaa vaikeasti tyhjää kappaletta? Tällaisissa tapauksissa on parempi kirjoittaa suomenkieliseen versioon jotain järkevää ja jättää muut kieliversiot pois. Tällöin Conflussakin kuvattu "käännösprosessi" huomaa puuttuvat käännökset. Muuten voisi tässäkin käydä niin, että nuo placeholderit menisivät sinällään tuotantoon, koska ne kuitenkin "näyttävät" ihan oikeilta käännöksiltä.

Reply:
Kieliversiot on nyt poistettu ja pistin tiketin kuvaus tekstin *.desc arvoksi.

Reply:
Btw. sen verran olin nähnyt tuon ruotsin kielisen käännöksen eteen vaivaan, että vilkaisin mitä termejä Finlex käyttää ja käytin samoja termeja. Jätetäänpä kuitenkin kääntäjälle kun mulla ruotsin määräisten ja epämääräisten muotojen kanssa on aina arpomista, ja prepositiotkin menevät vain ajoittain oikein.